src/components/molecules/MenuListItem/MenuListItem.tsx

```diff
@@ -0,0 +1,44 @@
+import { useMemo } from "react";
+import { ChevronDownIcon } from "@heroicons/react/24/outline";
+import {
+  MENU_LIST_ITEM_CONTAINER_CLASS_NAME,
+  MENU_LIST_ITEM_DESCRIPTION_CLASS_NAME,
+  MENU_LIST_ITEM_ICON_CLASS_NAME,
+  MENU_LIST_ITEM_TEXT_CLASS_NAME,
+} from "./const";
+import type { MenuListItemProps } from "./type";
+
+export const MenuListItem = ({
+  type = "dropdown",
+  chevron = true,
+  description,
+  children,
+  icon,
+  selectedIcon,
+  disabled,
+  selected = false,
+  onClick,
+}: MenuListItemProps) => {
+  const iconUI = useMemo(() => {
+    if (selected) {
+      return selectedIcon || icon;
+    }
+
+    return icon;
+  }, [icon, selected, selectedIcon]);
+
+  return (
+    <button
+      className={MENU_LIST_ITEM_CONTAINER_CLASS_NAME({ selected, type })}
+      disabled={disabled}
+      onClick={onClick}
+    >
+      {icon && <span className={MENU_LIST_ITEM_ICON_CLASS_NAME({ selected })}>{iconUI}</span>}
+      <div className="flex flex-col items-start">
+        <span className={MENU_LIST_ITEM_TEXT_CLASS_NAME({ selected })}>{children}</span>
+        <span className={MENU_LIST_ITEM_DESCRIPTION_CLASS_NAME({ selected })}>{description}</span>
```

Comment:
I think it would be better to use optional chaining to display the `description` or not:
```suggestion
        {description.length > 0 && <span className={MENU_LIST_ITEM_DESCRIPTION_CLASS_NAME({ selected })}>{description}</span>}
```
Otherwise the `span` will always render, whether there is a description or not.